src/_P050_TCS34725.ino

```diff
@@ -106,16 +108,16 @@ boolean Plugin_050(byte function, struct EventStruct *event, String& string)
         String optionsRGB[P050_RGB_OPTIONS];
         optionsRGB[0] = F("Raw RGB");
         optionsRGB[1] = F("Normalized RGB (0..255)");
-        optionsRGB[2] = F("Normalized sRGB (0.00000..1.00000)");
+        optionsRGB[2] = F("Normalized rgb (0.00000..1.00000)");
```

Comment:
Why is RGB here in lower case?
You also seem to differentiate between upper- and lower case in the note below.

Reply:
That was a suggestion I took from @heinemannj in the linked issue #3453 but I'm not sure I like it that way, sRGB does look clearer to me. I'm not familiar with the custom abbreviations in this color-based field.

Reply:
sRGB is a color space definition.
![image](https://user-images.githubusercontent.com/3751318/104107224-00294a00-52bb-11eb-83c8-aefc965757b8.png)
And the range (0....1) is an X/Y coordinate in the CIE1931 diagram. (N.B. it is 2-dimensional)

See also [this post on Eizo.nl](https://www.eizo.nl/praktische-informatie/knowhow-over-monitoren/kennis-over-kleurruimtes/) (not read it, but looks to go into details at first glance)

Reply:
> looks to go into details at first glance

It does go deeper then I want to read into right now 😛.
Guess I'll be reverting rgb to sRGB then, np.

Reply:
- RGB is a color system / model
- sRGB is an RGB model defined by Microsoft
- rgb are normalized RGB values which are used for further transformation into other color systems

http://www.brucelindbloom.com/index.html?ColorCalculator.html

Please crosscheck: https://medium.com/hipster-color-science/a-beginners-guide-to-colorimetry-401f1830b65a

![image](https://user-images.githubusercontent.com/3251642/104114535-d5181800-5305-11eb-94a4-34c083ff591b.png)

![image](https://user-images.githubusercontent.com/3251642/104114477-2b388b80-5305-11eb-8f35-9723aa86181a.png)

Reply:
```
          case 2:
            if (t != 0) { // r/g/b (normalized to 0.00..1.00 (but avoid divide by 0)
              UserVar[event->BaseVarIndex]     = (float)r / t * 1.0f * calRed;
              UserVar[event->BaseVarIndex + 1] = (float)g / t * 1.0f * calGreen;
              UserVar[event->BaseVarIndex + 2] = (float)b / t * 1.0f * calBlue;
            }
            break;
```
You are right - If you apply calibration on RGB values you have to apply the same calibration on rgb values too.

But on the other hand rgb are normalized/scaled RGB values which are only used for further transformation into other color systems.

=>rgb values from normal user perspective are not really relevant and reviewing our discussion this kind of redundant data/information potentially open recurring discussions and questions.

If somebody needs to transform RGB values into other color spaces he can calculate rgb values from the calibrated RGB values by himself or can use a professional Color Calculator
http://www.brucelindbloom.com/index.html?ColorCalculator.html

The most important exercise was to fix the Adafriut bug for normalized RGB calculation - meaning not to divide through the Clear channel but to divide through to sum of RBG values that the Sum of rgb will remain 1.

My conclusion:
- We can securely remove the redandant rgb part without losing any precision and information

Sorry for the additional work and time but I'm also on a long journey to understand Colorimetry.

Reply:
A lot of color spaces do take the color in 2 dimensions and the luminocity in a 3rd, precisely for this reason.
For example YUV and HSV are such color spaces.

And to give a very short introduction to the horse shoe shaped curve you see in the CIE1931 color space...
The outer curved line (not the straight line at the bottom) represent colors of a single wave length (typically reproducible using lasers)
This means the colored part of the chart is the range of colors we humans can see.

![image](https://user-images.githubusercontent.com/3751318/104121995-aec49d80-5342-11eb-9b5f-7f7a720ef9d1.png)

If you take 3 (or more) points on that chart (regardless whether they are inside or outside the chart), you can create all colors within that area of those points by mixing light (this chart is about light, not paint!!).
So to get back of the sRGB and Adobe color spaces (and there are more defined like NTSC, etc.) you see they are defined using coordinates in the XY plane of that chart.
Meaning to create a sRGB monitor, you should use primary colors of exactly those coordinates.
That's nearly impossible, so that's why monitor specs often refer to NN% of some color space when they mention the gamut (= the number of colors you can reproduce)

Typically "white" is around (1/3 , 1/3), but you also often see this chart with a curved line indicating a range of "daylight colors", like 5500K for example.
![image](https://user-images.githubusercontent.com/3751318/104122047-fea36480-5342-11eb-804c-a4a3a2dad307.png)

N.B. the blue numbers around the monochromatic curve are wavelength in nm.

To get an idea of what "infinite" color temperature looks like...
That's just about the color of the sky at a very cold sunny winter day.